simapp/export.go

```diff
@@ -80,7 +79,10 @@ func (app *SimApp) prepForZeroHeightGenesis(ctx sdk.Context, jailAllowedAddrs []
 		if err != nil {
 			panic(err)
 		}
-		_, _ = app.DistrKeeper.WithdrawValidatorCommission(ctx, valBz)
+		_, err = app.DistrKeeper.WithdrawValidatorCommission(ctx, valBz)
```

Comment:
I don't think we should check the error, it fails tests when a validator has no commission.
Could you revert?